doc/index.rst

```diff
@@ -432,6 +432,30 @@ Example of returned tag data:
 
         ['Authority in literature', 'Errata']
 
+===================
+Retrieving Version Information
+===================
+
+The Zotero API recommends requesting version information for all (or all changed) items and collections when implementing syncing.  The following convience methods simplify this process.  Note that by default these methods impose no limit on the number of items/collection versions returned.  To retrieve only the versions of items changed after a given version use the since search/request parameter.
```

Comment:
Sure, it is covered briefly in your stuff on search parameters but it could be more clear.  I'll make the change.